pkcs12.go

```diff
@@ -646,9 +655,39 @@ func (enc *Encoder) Encode(privateKey interface{}, certificate *x509.Certificate
 	if localKeyIdAttr.Value.Bytes, err = asn1.Marshal(certFingerprint[:]); err != nil {
 		return nil, err
 	}
+	pkcs12Attributes := []pkcs12Attribute{localKeyIdAttr}
+
+	//asn1 values are in line with the implementation in EncodeTrustStoreEntries
```

Comment:
turbo nitpick: I think some linters and formatters would reject this because there's no space between the `//` and the text. We don't really have linting and stuff on this repo and I know this is super annoying but I think we should tweak if we can:

```suggestion
	// ASN.1 values are in line with the implementation in EncodeTrustStoreEntries
```